servicetalk-traffic-resilience-http/src/main/java/io/servicetalk/traffic/resilience/http/AbstractTrafficResilienceHttpFilter.java

```diff
@@ -280,6 +287,27 @@ public void cancel() {
                         }
                     }
                 }, true));
+        if (ENABLE_DEBUGGING) {
+            response = response.liftSync(subscriber -> new SingleSource.Subscriber<StreamingHttpResponse>() {
```

Comment:
Because we have `BeforeFinallyHttpOperator` with `discardEventsAfterCancel=true` above, it will be better if we place this before `BeforeFinallyHttpOperator` to make sure we are seeing the same events as the ticket will see.